MAUI/Calendar/migration.md

```diff
@@ -778,7 +778,7 @@ this.Content = calendar;
 </tr>
 <tr>
```

Comment:
YearCellLoadedEventArgs | Nil | Not Supported.
-- | -- | --


[YearCellLoadedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.YearCellLoadedEventArgs.html)	Nil	Not Supported.

Viewchanged event args can apply?